pyproject.toml

```diff
@@ -56,16 +56,19 @@ python-tag = "py3"
 [project.optional-dependencies]
 duckdb = ["fugue[duckdb]"]
 spark = ["pyspark[connect]>=3.1.1; python_version < \"3.11\"", "pyspark[connect]>=3.4; python_version >= \"3.11\""]
+snowflake = ["snowflake-connector-python", "snowflake-snowpark-python"]
```

Comment:
[NON-BLOCKING]: how does `snowflake-snowpark-python` affect the dependency tree? Does it add a bunch of packages to the `snowflake` extra?

Reply:
I think it most depends on the snowflake-connector-python if you look here: https://github.com/snowflakedb/snowpark-python/blob/main/setup.py#L16